src/components/RegistrationErrorActions.tsx

```diff
@@ -43,7 +41,7 @@ export const RegistrationErrorActions = ({
         variant="outlined"
         component={Link}
         size="small"
-        to={getRegistrationWizardPath(registrationIdentifier, firstErrorTab)}
+        to={getWizardPathByRegistration(registration, { tab: firstErrorTab })}
```

Comment:
Denne, og kanskje noen flere må ha `state={{ previousPath: window.location.pathname } satisfies PreviousPathLocationState}` for at det skal virke når man trykker "Lagre og se".

Ser på å fikse det der det trengs